.github/workflows/fastcdr-test.yml

```diff
@@ -60,13 +60,32 @@ jobs:
       with:
           path: src/Fast-CDR
 
-    - name: Install Colcon and dependencies
+    - name: Install Colcon and Dependencies
       uses: eProsima/eProsima-CI/ubuntu/install_colcon@main
 
+    # Temporal step as this platform will be discontinued soon
+    - name: Install Gtest Backwards Compatibility
+      if: ${{ matrix.ubuntu-version == 'ubuntu-20.04' }}
```

Comment:
Right now Ubuntu 22.04 is doing both installations. You could probably add a similar if clause to the other GTest installation.